### PR TITLE
chore(blog): add camunda-modeler-4.1 blog post

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Related to camunda/camunda-modeler#1861

### CODE REVIEW

File: content/post/2020/07/camunda-modeler-4.1-released.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+When working with complex DMN diagrams being able to navigate efficiently is crucial. We introduced a new means of navigating the DRD diagram elements without having to go back and forth between DRD and DRG element view.
+
+{{< figure class="no-border teaser" src="overview.png" alt="DMN Overview Navigation" caption="DMN Overview Navigation" >}}
+
+When working on a single decision table or literal expression, you can resize the overview or close hide it.
+
+## Usage Statistics
+
+If you want to help us make the Camunda Modeler an even better product, you can now enable it to send usage statistics. Note that we do not track any personal or identifiable data. No statistics will be sent without your permission. Go to `Help > Privacy Preferences` to enable and disable this feature.
```
Comment (philippfromme) [inline content/post/2020/07/camunda-modeler-4.1-released.md:40]: Maybe change to _No data will be sent without your permission and no sensitive data will be sent ever._

Comment (pinussilvestrus) [reply content/post/2020/07/camunda-modeler-4.1-released.md]: That sounds even better

Comment (philippfromme) [reply content/post/2020/07/camunda-modeler-4.1-released.md]: ❤️

File: content/post/2020/07/camunda-modeler-4.1-released.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+We are excited to announce the release of Camunda Modeler v4.1. This release introduces a new layout to the DMN decision table editor and a more efficient way to navigate DMN diagrams.
+
+[Download the latest release](https://camunda.com/download/modeler/) and start modeling right away.
+
+<!--more-->
+
+## New Decision Table Layout
+
+Being able to reason about your decision tables without having to dig into details is crucial. As part of our ongoing effort to make the DMN editor an even better tool, we're introducing changes to the decision table layout to increase overall readability and, therefore, understandability.
```
Comment (MaxTru) [inline content/post/2020/07/camunda-modeler-4.1-released.md:17]: Change from marco suggested to make it easier to understand:
Option A: _Being able to collaborate on a decision table without having to dig into detail is crucial_.
Option B: _Being able to collaborate and review a decision table without having to dig into detail is crucial_.

Comment (pinussilvestrus) [reply content/post/2020/07/camunda-modeler-4.1-released.md]: Chose Option B and integrated.

**Pull request merged**